### ISSUE TITLE
Unable to mention an opponent because it appears offline for Discord API

### ISSUE THREAD

**Issue by JoepTM** [utarwyn/discord-tictactoe#14]
I keep getting this message if i invite someone to duel with me:

> `You could not challenge [username]. Please mention another user.`

Could you please fix this

**Comment by utarwyn** [comment]
Hello @JoepTM, did you mentionned the opponent with **"@"**? Did you have an error in the console?

**Comment by JoepTM** [comment]
I mentioned with @ and i didn't get any errors in the console

**Comment by utarwyn** [comment]
By analyzing the source code, you can see all needed conditions needed by the bot to detect an opponent:
https://github.com/utarwyn/discord-tictactoe/blob/56cb8fd77a41bafcb813546a45dc7fab28629194/src/bot/commands/StartCommand.ts#L66-L72

Member must be **online**, **a real person**, and must has the permission to see the channel.
If you don't have error, I don't know how to help you more.

**Comment by JoepTM** [comment]
the member is online and a real person

:(

**Comment by JoepTM** [comment]
Something you probably need to know, i installed it with npm i didn't install the zip @utarwyn

**Comment by utarwyn** [comment]
Hmm indeed it does not seem to work for me aswell... I will try to fix that!

**Comment by JoepTM** [comment]
thanks

**Comment by utarwyn** [comment]
After few investigations, I have seen that presence of invited member is always set to **"offline"**, pretty strange. Maybe a bug of discord.js or the Discord API, I will do additional test to find out a solution.

My test user:
![image](https://user-images.githubusercontent.com/9255967/97791256-b93f1a00-1bd0-11eb-8f72-ef9719ada35d.png)

API response:
```
Presence {
  userID: '772220535546970142',
  guild: null,
  status: 'offline',
  activities: [],
  clientStatus: null
}
```

**Comment by utarwyn** [comment]
I understand the problem: Discord has changed the way to get member presences, this is now using privileged intents. Discord.js team has created an issue two days ago about it to explain that **Discord has made the use of this feature more restricted** : discordjs/discord.js#4961

To avoid these problems, I will remove the "offline check" for now. 
The API is becoming more and more restricted over time... very sad news.

**Anyway, sorry for the inconvenience.**

**Comment by utarwyn** [comment]
I have published a new version of the bot on NPM t**v2.0.7**) to fix the problem.
If your problem has been resolved, you can close the issue!

See it here: https://www.npmjs.com/package/discord-tictactoe/v/2.0.7

Best regards

**Comment by JoepTM** [comment]
thanks for solvinf the bug
